styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -92,6 +94,27 @@ public String createExecution(RunState runState, final String execName, final Fl
     }
   }
 
+  @Override
+  public void terminateExecution(RunState runState, FlyteExecutionId flyteExecutionId) {
+    requireNonNull(runState, "runState");
+    requireNonNull(flyteExecutionId, "flyteExecutionId");
+    try {
+      flyteAdminClient.terminateExecution(
+          flyteExecutionId.project(),
+          flyteExecutionId.domain(),
+          flyteExecutionId.name(),
+          TERMINATE_CAUSE);
+    } catch (StatusRuntimeException e) {
+      if (e.getStatus().getCode() == Status.Code.NOT_FOUND) {
+        LOG.warn("Trying to terminate non existent flyte execution: {}", flyteExecutionId);
+        return;
+      } // TODO: Consider other status codes that could be interesting
+      LOG.warn("Couldn't terminate flyte execution: {}", flyteExecutionId, e);
+    } catch (Exception e) {
+      LOG.warn("Couldn't terminate flyte execution: {}", flyteExecutionId, e);
```

Comment:
is it enough to simply log a warn here and move on?

Reply:
I will change the last one to log error but we will give an holistic review between styx and flyte failures in a different PR